custom_code/hooks.py

```diff
@@ -9,7 +11,18 @@ def target_post_save(target, created):
     """This hook runs following update of a target."""
     logger.info('Target post save hook: %s created: %s', target, created)
 
-    qprob, qso, qoffset = static_cats_query([target.ra], [target.dec])
+    qprob, qso, qoffset, asassnprob, asassn, asassnoffset = static_cats_query([target.ra], [target.dec])
```

Comment:
Does this now require a specific version of `kne_cand_vetting` to work? If so, you should specify that in the `requirements.txt`. Otherwise people may run into version problems when trying to run the updated TOM in an old environment.